packages/trader/src/Modules/Trading/Containers/trade.jsx

```diff
@@ -206,16 +243,14 @@ const ChartMarkers = connect(({ modules, ui, client }) => ({
 }))(Markers);
 
 class ChartTradeClass extends React.Component {
-    state = {
-        active_category: null,
-    };
     bottomWidgets = ({ digits, tick }) => <ChartBottomWidgets digits={digits} tick={tick} />;
     topWidgets = ({ ...props }) => {
-        const { is_digits_widget_active, try_synthetic_indices } = this.props;
+        const { is_digits_widget_active, try_synthetic_indices, try_open_markets, open_market } = this.props;
+
         return (
             <ChartTopWidgets
-                active_category={try_synthetic_indices ? 'synthetic_index' : null}
-                open={!!try_synthetic_indices}
+                open_market={open_market}
+                open={try_synthetic_indices || try_open_markets}
```

Comment:
so regards above apprach, we can change this to `open={open_market.category || open_market.subcategory || open_market.market}` or similar thing